.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
- [ ] Kindly fix the linter errors (stylelint) that are not passing at the moment 👍